.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+# Summary
+
+<<if there's an issue>>Fixes #issueNumber
```

Comment:
I would prefer to kill the `<<if there's an issue>>` text. People will already delete that if theres not.
maybe just `Closes #issueNumber`